### PR TITLE
[as4630-54pe]Modify to read correct psu SN and ID

### PR DESCRIPTION
**Pull request by jostar-yang**
**- Why I did it**
The PSU is specific. Get SN and ID is not meet with standard method.
Old code
```
# cat psu_serial_number
SB020X18195100004
(It lack the last one byte. It should be SB020X181951000046 )
as4630-54pe-2:/sys/bus/i2c/drivers/as4630_54pe_psu/11-0051# cat psu_model_name
YPEB1200?AM-1A03P10 ( It should beYPEB1200?AM )
```

**- How I did it**
Modift psu drv to get and parse/shift need offset to get correct data
**- How to verify it**
New Code
```
# cat psu_serial_number
SB020X181951000046

4630-54pe-2:/sys/bus/i2c/drivers/as4630_54pe_psu/11-0051# cat psu_serial_number
SB020X181951000047


as4630-54pe-2:/sys/bus/i2c/drivers/as4630_54pe_psu/11-0051# cat psu_model_name
YPEB1200AM
as4630-54pe-2:/sys/bus/i2c/drivers/as4630_54pe_psu/11-0051#
as4630-54pe-2:/sys/bus/i2c/drivers/as4630_54pe_psu/11-0051#
4630-54pe-2:/sys/bus/i2c/drivers/as4630_54pe_psu/11-0051# cat psu_serial_name
SB020X181951000047
as4630-54pe-2:/sys/bus/i2c/drivers/as4630_54pe_psu/10-0050# cat psu_model_name
YPEB1200AM
```

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-accton/as4630-54pe/modules/x86-64-accton-as4630-54pe-psu.c [hunk 1]
```diff
@@ -286,21 +284,36 @@ static struct as4630_54pe_psu_data *as4630_54pe_psu_update_device(struct device
...
             {
                 data->serial_number[0] = '\0';
-                dev_dbg(&client->dev, "unable to read model name from (0x%x) offset(0x2e)\n", client->addr);
-                printk("unable to read model name from (0x%x) offset(0x2e)\n", client->addr);
+                dev_dbg(&client->dev, "unable to read model name from (0x%x) offset(0x35)\n", client->addr);
             }
+            if (!strncmp(data->model_name, "YPEB1200AM", strlen("YPEB1200AM"))) /*for YPEB1200AM, SN length=18*/
+            {
             data->serial_number[MAX_SERIAL_NUMBER-1]='\0';
+        }
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-accton/as4630-54pe/modules/x86-64-accton-as4630-54pe-psu.c:313]: These lines are not aligned. Need to be indented 4 more spaces.

**Review by jleveque — CHANGES_REQUESTED** [review]

**Comment by jleveque** [comment]
Retest mellanox please

**Comment by jleveque** [comment]
Retest vsimage please

**Review by jleveque — DISMISSED** [review]

**Comment by lguohan** [comment]
Retest vsimage please

**Comment by lguohan** [comment]
Retest mellanox please

**Comment by jostar-yang** [comment]
Retest vsimage please

**Comment by jostar-yang** [comment]
Can you merge code?　This PR  pass all checking for a long time.

**Review by lguohan — APPROVED** [review]

**Pull request merged**